Bottom: 8366a50fa5ca152bee136892dff4bc184f12b4cd
Top:    a67f701fbc904bcaea2ce884e38088db7feaff9d
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-06-27 14:58:01 -0700

Refresh of shut_wr

---

diff --git a/src/rsocket.c b/src/rsocket.c
index e3da27f..419e9df 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -1137,7 +1137,7 @@ ssize_t rrecv(int socket, void *buf, size_t len, int flags)
 	fastlock_acquire(&rs->rlock);
 	if (!rs_have_rdata(rs)) {
 		ret = rs_get_comp(rs, rs_nonblocking(rs, flags), rs_conn_have_rdata);
-		if (ret) // && errno != ECONNRESET)
+		if (ret)
 			goto out;
 	}
